caravel/templates/index.html

```diff
@@ -3,11 +3,33 @@
 {% block head %}
   {{ super() }}
   <script src="/static/js/continuation.js" type="text/javascript"></script>
+  <script src="/static/js/view.js" type=text/javascript></script>
 {% endblock %}
 {% block content %}
-  <div class="row" id="listings">
-    {% for listing in listings %}{% include "listings/inline.html" %}{%
-      endfor %}
+  <div class = "row">
+    <button type="button" class="btn btn-default">
```

Comment:
Use just a button, not a linky-button-inna-button (i.e. `<a class="btn btn-default">` instead of `<button class="btn btn-default"><a>`). That also avoids adding another style!